mod.ts

```diff
@@ -3,8 +3,14 @@ import { readFileStr } from "https://deno.land/std/fs/read_file_str.ts";
 import { StringReader } from "https://deno.land/std/io/readers.ts";
 import { BufReader } from "https://deno.land/std/io/bufio.ts";
 
+export interface HeaderOption {
+  name: string;
+  parse?: (string) => unknown;
```

Comment:
This is implicitly of an `any` type. Did you mean this instead?

```suggestion
  parse?: (value: string) => unknown;